src/colorizer/ColorizeCanvas3D.ts

```diff
@@ -181,15 +205,16 @@ export class ColorizeCanvas3D implements IRenderCanvas {
     });
     this.view3d.addVolume(volume);
 
-    this.view3d.setVolumeChannelEnabled(volume, 0, true);
-    this.view3d.setVolumeChannelOptions(volume, 0, {
+    const segChannel = this.params.dataset?.segmentationChannel ?? 0;
+    this.view3d.setVolumeChannelEnabled(volume, segChannel, true);
```

Comment:
necessary?

Reply:
Depends on if there are other channels in the segmentation volume. AICS typically only has the segmentation volume, but it's not guaranteed, right?